src/stan/analyze/mcmc/autocovariance.hpp

```diff
@@ -0,0 +1,114 @@
+#ifndef STAN_ANALYZE_MCMC_AUTOCOVARIANCE_HPP
+#define STAN_ANALYZE_MCMC_AUTOCOVARIANCE_HPP
+
+#include <stan/math/prim/mat/fun/Eigen.hpp>
+#include <stan/math/prim/mat.hpp>
+#include <unsupported/Eigen/FFT>
+#include <complex>
+#include <vector>
+
+namespace stan {
+namespace analyze {
+
+  /**
+   * Write autocorrelation estimates for every lag for the specified
+   * input sequence into the specified result using the specified FFT
+   * engine.  The return vector will be resized to the same length as
+   * the input sequence with lags given by array index.  Uses "biased"
+   * estimate as recommended by Geyer (1992).
```

Comment:
There are multiple Geyer (1992) references.  Can you include the permalink, https://projecteuclid.org/euclid.ss/1177011137, or the DOI, doi:10.1214/ss/1177011137, here and below to avoid any ambiguity?

Reply:
Pedantic -- I would be a bit more explicit and say something like "Normalizes lag-k autocovariance estimators by N instead of (N - k), yielding biased but more stable estimators as discussed in Geyer (1992)".